lib/screens/explore_screens/vouch/vouched_tab/components/not_qualified_to_vouch_dialog.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/components/custom_dialog.dart';
+import 'package:seeds/images/explore/vouch_white_background.dart';
+
+class NotQualifiedToVouchDialog extends StatelessWidget {
+  const NotQualifiedToVouchDialog({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return CustomDialog(
+      icon: const CustomPaint(size: Size(60, 60), painter: VouchWhiteBackground()),
+      singleLargeButtonTitle: "Ok, Thank you!",
+      onSingleLargeButtonPressed: () {
+        Navigator.of(context).pop();
+      },
```

Comment:
No needed to pass pop, this is the default behavior of CustomDialog's onSingleLargeButtonPressed.

Reply:
fix